README.md

```diff
@@ -60,3 +60,35 @@ npmlock2nix.node_modules {
 }
 ```
 
+# Contributing
```

Comment:
I would like to suggest to break this up a little

## Contributing
Contributions to the project are welcome in the form of GitHub PRs. Please consider the following guidelines before creating PRs:

- Please make sure your Nix code is formatted (see ..formatting..)
- Please try to provide tests for the changes or features your PR introduces (see ..testing..)
- If you are planning to make any considerable changes, please consider first presenting them in a GitHub issue so they can be discussed

### Formatting
### Testing